### ISSUE TITLE
Add probabilities when user does not know previous pattern

### ISSUE THREAD

**Issue by ColMelvin** [mikebryant/ac-nh-turnip-prices#68]
We can calculate the steady state probability of any given pattern using a [Markov model](https://en.wikipedia.org/wiki/Markov_model).  Thus, we can get _initial_ probabilities for the 4 states and display it to the users when they select "I don't know" for the "Previous Pattern".  This builds on issue #13.

If my calculations are correct (details below; someone please verify), given no information about the previous week and no information on Mon - Sat for the current week, the chances for each pattern are:
```
pattern0: 0.346
pattern1: 0.247
pattern2: 0.148
pattern3: 0.259
```

I'm using a [Markov chain](https://en.wikipedia.org/wiki/Markov_chain) with the pattern numbers & probabilities from [Ninji's `TurnipPrices::calculate()`](https://gist.github.com/Treeki/85be14d297c80c8b3c0a76375743325b#file-turnipprices-cpp-L135-L210).  And, since I've not done these calculations in over a decade, I'm using an [online calculator](https://kilin.clas.kitasato-u.ac.jp/software/markov/markov.html) with the following data:
```
0.20 0.30 0.15 0.35
0.50 0.05 0.20 0.25
0.25 0.45 0.05 0.25
0.45 0.25 0.15 0.15
```

I'm not certain how this is affected by issue #48, which seeks to update probabilities based on Bayes.  I suspect the problems are independent; that is, the solution for issue #48 will just work when given these probabilities.  If not, maybe a [HMM](https://en.wikipedia.org/wiki/Hidden_Markov_model) would work?

**Comment by peter50216** [comment]
> If my calculations are correct (details below; someone please verify), given no information about the previous week and no information on Mon - Sat for the current week, the chances for each pattern are:
> 
> ```
> pattern0: 0.346
> pattern1: 0.247
> pattern2: 0.148
> pattern3: 0.259
> ```

Verified that these numbers looks good to me: [Wolfram alpha link](https://www.wolframalpha.com/input/?i=Eigenvector%5BTranspose%5B%7B%7B0.2%2C+0.3%2C+0.15%2C+0.35%7D%2C+%7B0.5%2C+0.05%2C+0.2%2C+0.25%7D%2C+%7B0.25%2C+0.45%2C+0.05%2C+0.25%7D%2C+%7B0.45%2C+0.25%2C+0.15%2C+0.15%7D%7D%5D%5D), [link 2](https://www.wolframalpha.com/input/?i=%7B1.33826%2C+0.955982%2C+0.570458%2C+1%7D%2FSum%5B%7B1.33826%2C+0.955982%2C+0.570458%2C+1%7D%5D) (The vector in link 2 is the first eigenvector for the previous link)

> I'm not certain how this is affected by issue #48, which seeks to update probabilities based on Bayes. I suspect the problems are independent; that is, the solution for issue #48 will just work when given these probabilities. If not, maybe a [HMM](https://en.wikipedia.org/wiki/Hidden_Markov_model) would work?

I also think that #48 should just work with these probabilities.

**Comment by ZZZZzzzzac** [comment]
hasn't verify yet, but issue #48 can work with this just fine, it only provides another pre-defined probability pattern. Use these pre-defined probabilities as P(pattern is k) after entering MonAM price

**Comment by ColMelvin** [comment]
@peter50216 Thank you for validating and, in particular, for providing the Wolfram Alpha links.  They helped greatly with developing a solution.

**Comment by ColMelvin** [comment]
Fixed in #109.
